feat: DIA-1862: Improve SDK coverage and data flow

robot-ci-heartex opened the pull request:
Hi @robot-ci-heartex!

This PR was automaticaly generated via Follow Merge. 
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.

- https://github.com/HumanSignal/label-studio-sdk/pull/403 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/13024134862)

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13024134862

Commit:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13024136341

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13024234400

Commit:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13024253291

Commit:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13024326092

Commit:
Add edition type in /version

Commit:
Add tests

Commit:
Fix tests

Commit:
Update tests

Commit:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13039643166

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13039664311

Commit:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13039820375

matt-bernstein commented on label_studio/tests/sdk/fixtures.py:
```diff
@@ -0,0 +1,29 @@
+
+
+@pytest.fixture
+def test_project_with_view(django_live_url, business_client):
```
Makes sense to have these as fixtures now with the current way we use pytest, but looking forward to progressively replacing them with something like [Factory Boy](https://www.notion.so/humansignal/Testing-Standardization-Proposal-15a7094e54c880ed8d0ce4078c666641?pvs=4#15b7094e54c880ae98efcd905a4fd0b5) when available (cc @mcanu )

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13100164909

robot-ci-heartex committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13100231038

niklub commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13102671774)
> Successfully merged:  38 files changed, 247 insertions(+), 208 deletions(-)

niklub committed:
Merge branch 'develop' into 'fb-dia-1862'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13102671774
